client-side/src/stories/Select/select.scss

```diff
@@ -4,7 +4,8 @@
 
   .genericSelect {
     &.primary {
-      background-color: $background-class;
+
+      background-color: white;
```

Comment:
you can't use colors in this file, it should be imported from variables file.
also, notice that if you change the color here it will affect all usages of select in all the site